pkg/project/version_bundle_azure.go

```diff
@@ -7,11 +7,19 @@ import (
 var versionBundleAzure = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
-			Component:   "cluster-operator",
-			Description: "TODO",
+			Component:   "nginx-ingress-controller",
+			Description: "Supported enabling the proxy protocol with AWS ELB.",
```

Comment:
Not sure should this AWS relevant change be mentioned in Azure changelog

Reply:
Can be omitted for both non affected providers IMO. An upgrade notice is in there - that should be sufficient.

Reply:
Thanks, I removed it.